install.sh

```diff
@@ -647,7 +647,7 @@ killtree() {
 }
 
 getshims() {
-    ps -e -o pid= -o args= | sed -e 's/^ *//; s/\s\s*/\t/;' | grep -w 'k3s/data/[^/]*/bin/containerd-shim' | cut -f1
+    ps -e -o pid= -o args= | sed -e 's/^ *//; s/\s\s*/\t/;' | grep -w 'containerd-shim.*-namespace k8s.io' | cut -f1
```

Comment:
this sed is gnu specific, is that a concern?

Reply:
The sed expression isn't changing, is it? Just the grep.